integrations/getServices/index.test.ts

```diff
@@ -0,0 +1,14 @@
+import { ensure as ensureEnvValue } from "../../lib/env";
+import {
+  testInvalidToken,
+  testNoToken
+} from "../../lib/helpers/testUnauthorized";
+
+describe("getServices", () => {
+  const host = ensureEnvValue("IO_BACKEND_HOST");
```

Comment:
we have a side-effet here (`throw`) which is implicit. try to never throw or make the fact that you're throwing explicit. it's a complex task in JS to know in advance which methods may throw or not. prefer using eithers / taskeithers.

Reply:
I can just place the `getOrElseL` here. Is duplication preferable in this case?

Reply:
that's ok to put `getOrElseL` here